### PR TITLE
st1 [1790][ADD] website_sale_template_adj

### PR DESCRIPTION
**Pull request by kakurai8**
[1790](https://www.quartile.co/web?debug=1#id=1790&action=771&model=project.task&view_type=form&menu_id=505)

- Ref. odoo>addons>website_sale>views>templates.xml

### CODE REVIEW

File: website_sale_templates_view_adj/views/website_sale_templates_view_adj.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        <xpath expr="//div[@id='payment_method'][hasclass('js_payment')]" position="after">
+            <div class="custom-control custom-checkbox float-right mt-2 oe_accept_cgv_button">
+                <input type="checkbox" id="checkbox_cgv" class="custom-control-input"/>
+                <label for="checkbox_cgv" class="custom-control-label">
+                    We agree to the <a target="_BLANK" href="/shop/terms">terms &amp; conditions</a>
+                    <br/>
+                    according to the current agreement.
+                </label>
+            </div>
+        </xpath>
```
Comment (yostashiro) [inline website_sale_templates_view_adj/views/website_sale_templates_view_adj.xml]:
```suggestion
        <xpath expr="//label[hasclass('custom-control-label')]" position="after">
            <span>
                according to the current agreement.
            </span>
        </xpath>
```

Note that this style of embedding tags in the middle of a sentence makes the translation tricky. It should be OK this time as it's intended to be enable for the overseas website (i.e. English) only.

**Review by yostashiro — COMMENTED** [review]
@kakurai8 Please change the module name to website_sale_template_adj, and go through the usual steps of generating README, etc.

File: website_sale_template_adj/views/website_sale_template_adj.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    <template
+        id="website_sale_templates_view_adj"
+        inherit_id="website_sale.payment_sale_note"
+    >
+        <xpath expr="//label[hasclass('custom-control-label')]" position="after">
+            <span>
+                <br />
+                according to the current agreement.
+            </span>
+        </xpath>
```
Comment (tack-yabeQuartile) [inline website_sale_template_adj/views/website_sale_template_adj.xml:12]:
```xml
        <xpath expr="//label[hasclass('custom-control-label')]" position="attributes">
            <attribute name="invisible">1</attribute>
        </xpath>
        <xpath expr="//input[hasclass('custom-control-input')]" position="after">
            <label for="checkbox_cgv" class="custom-control-label">
                We agree to the <a target="_BLANK" href="/shop/terms">terms &amp; conditions</a>
            </label>
            <span>
                <br />
                according to the current agreement.
            </span>
        </xpath>
```

File: website_sale_template_adj/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+{
+    "name": "eCommerce: View Adjustments",
+    "summary": "",
```
Comment (yostashiro) [inline website_sale_template_adj/__manifest__.py]:
```suggestion
```

File: website_sale_template_adj/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    "version": "12.0.1.0.1",
+    "category": "Website",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "license": "LGPL-3",
+    "application": False,
```
Comment (yostashiro) [inline website_sale_template_adj/__manifest__.py]:
```suggestion
```

File: website_sale_template_adj/views/website_sale_template_adj.xml (hunk 2 above)
Comment (tack-yabeQuartile) [reply website_sale_template_adj/views/website_sale_template_adj.xml:12]: I suggest this. "I -> We"

File: website_sale_template_adj/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Quartile Limited
```
Comment (tack-yabeQuartile) [inline website_sale_template_adj/__manifest__.py]: 2021

File: website_sale_template_adj/controllers/__init__.py [hunk 6]
```diff
@@ -0,0 +1 @@
+from . import main
```
Comment (yostashiro) [inline website_sale_template_adj/controllers/__init__.py]: Delete this file and the controllers folder.

File: website_sale_template_adj/readme/CONTRIBUTORS.rst [hunk 7]
```diff
@@ -0,0 +1 @@
+* Quartile Limited <info@quartile.co>
```
Comment (yostashiro) [inline website_sale_template_adj/readme/CONTRIBUTORS.rst]: No need to have this file for our custom modules.

File: website_sale_template_adj/readme/DESCRIPTION.rst [hunk 8]
```diff
@@ -0,0 +1 @@
+This module adjusts the view when customers pay on the website.
```
Comment (yostashiro) [inline website_sale_template_adj/readme/DESCRIPTION.rst]:
```suggestion
This module makes adjustments to eCommerce templates.
```

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Review by tack-yabeQuartile — DISMISSED** [review]

File: website_sale_template_adj/__manifest__.py (hunk 5 above)
Comment (kakurai8) [reply website_sale_template_adj/__manifest__.py]: Thank you!

File: website_sale_template_adj/readme/CONTRIBUTORS.rst (hunk 7 above)
Comment (kakurai8) [reply website_sale_template_adj/readme/CONTRIBUTORS.rst]: I got it!

File: website_sale_template_adj/controllers/__init__.py (hunk 6 above)
Comment (kakurai8) [reply website_sale_template_adj/controllers/__init__.py]: I got it!

File: website_sale_template_adj/views/website_sale_template_adj.xml (hunk 2 above)
Comment (kakurai8) [reply website_sale_template_adj/views/website_sale_template_adj.xml:12]: - We tried also bellow, but it doesn't work as expected.
- Once, I'll update with minimal adjustments.

       <xpath expr="//div[hasclass('custom-control')]" position="attributes">
            <attribute name="class">hidden</attribute>
        </xpath>
        <xpath expr="//input[hasclass('custom-control-input')]" position="after">
            <label for="checkbox_cgv" class="custom-control-label">
                 We agree to the <a target="_BLANK" href="/shop/terms">terms & conditions</a>
            </label>
                <span>
                <br />
                    according to the current agreement.
                </span>
        </xpath>

File: website_sale_template_adj/__manifest__.py [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Quartile Limited
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+{
+    "name": "eCommerce: View Adjustments",
+    "version": "12.0.1.0.1",
```
Comment (tack-yabeQuartile) [inline website_sale_template_adj/__manifest__.py]: I suggest "12.0.1.0.0" because it is still unpublished module. (just comment.)

Comment (kakurai8) [reply website_sale_template_adj/__manifest__.py]: You're right!
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers

**Review by tack-yabeQuartile — APPROVED** [review]

**Pull request merged**